lib/modules/platform/github/index.md

```diff
@@ -1,5 +1,12 @@
 # GitHub and GitHub Enterprise
 
+## Authentication
+
+First, [create a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the bot account (select "repo" scope).
+Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+
+For GitHub Enterprise Server set the `endpoint` in your `config.js` to `https://github.enterprise.com/api/v3/`.
```

Comment:
Why do we insist that this must be set in `config.js`? It can also be set via envvars or CLI params.